app/src/test/java/me/saket/dank/utils/UrlParserTest.java

```diff
@@ -480,6 +480,17 @@ public void parseStreamableUrl() {
     assertThat(((StreamableUnresolvedLink) parsedLink).videoId()).isEqualTo("jawcl");
   }
 
+  @Test
+  public void parseYouTubeUrl() {
+    String url = "https://www.youtube.com/watch?v=3tCBIDzF34M";
+    assertEquals(UrlParser.isYouTubeUrl(url), true);
```

Comment:
It looks to me like this won't pass. The `isYouTubeUrl` accepts only host (which is not clear from the name) and you pass whole url. Should either rename the method and pass only host or modify it to work with whole urls.